source/semver.d

```diff
@@ -395,7 +392,7 @@ struct SemVer
     }
 
     /// ditto
-    VersionPart differAt(const SemVer other) const scope @safe pure nothrow
+    VersionPart differAt(const SemVer other) const scope @safe pure
```

Comment:
Unrelated ?

Reply:
Throwing a assert is not nothrow. 
Surely unreleated. I can splitt the PR into 3 separate.

You can check the latest DMD beta build, and think about, if DMD is wrong or the current code without the PR.

May intention is to get rid of the warnings.